backend/main.go

```diff
@@ -71,9 +72,16 @@ type Post struct {
 	CreatedAt string `db:"created_at" json:"created_at"`
 }
 
+var Index = 0
+
 func (h *Handler) GetPosts(c echo.Context) error {
+	pageParam := c.QueryParam("page")
+	page, _ := strconv.Atoi(pageParam)
+
+	Index := (page) * 20
```

Comment:
変数名は camelCase が推奨されているので、 `index` に変更してほしいです！
それと `page` には括弧をつける必要はないので外すと読みやすくなってうれしいです

```suggestion
	index := page * 20
```